pkg/tools/clienturlctx/context.go

```diff
@@ -36,6 +38,7 @@ func WithClientURL(parent context.Context, clientURL *url.URL) context.Context {
 	if parent == nil {
 		panic("cannot create context from nil parent")
 	}
+	log.FromContext(parent).Infof("passed clientURL: %v", clientURL)
```

Comment:
Could we make this Debug not Info level?

Reply:
Thanks, fixed!